src/constants/global.constants.ts

```diff
@@ -1,4 +1,15 @@
 export const json_provider: string = 'https://testnet.zkevm.cronos.org';
-export const tokenAddress: string = '0x66c8221E5938A296D7542aB3D7cB856e789C06B2';
-export const routerAddress: string = '0x67876A7a191df6E9610B069Bd7f1b8cC1a7fD20D';
-export const wrappedZkcroAddress: string = '0xf9Bb37013de8Cd3f89b3623Af9EE1b1B32D872C9';
+
+export const ROUTER_MAPPING: Record<number, string> = {
+  25: '0x66C0893E38B2a52E1Dc442b2dE75B802CcA49566',
+  338: '0xC74C960708f043E04a84038c6D1136EA7Fcb16a1',
+  388: '0x4E792B8c9bcB9E200C3713810C4D6eA8C4230E7c',
+  240: '0x9EB4db2E31259444c5C2123bec8B17a510C4c72B',
+};
+
+export const WRAPPED_ADDRESS_MAPPING: Record<number, string> = {
+  25: '0x5C7F8A570d578ED84E63fdFA7b1eE72dEae1AE23', // Mainnet WCRO
+  338: '0x6a3173618859c7cd40faf6921b5e9eb6a76f1fd4', // Testnet WCRO
+  388: '0xC1bF55EE54E16229d9b369a5502Bfe5fC9F20b6d', // Mainnet wzkCRO
+  240: '0xeD73b53197189BE3Ff978069cf30eBc28a8B5837', // Testnet wzkCRO
+};
```

Comment:
just suggestion, can we check validity of contract,
If address is wrong, show more detailed error message like

`check contract address` ?

Reply:
can be done as another issue

Reply:
Good point, maybe it can be an optimisation point for signer-app -> optimise error handling for frontend